pkg/resourcecreator/pod/pod.go

```diff
@@ -208,8 +209,13 @@ func fromEnvConfigmap(name string) corev1.EnvFromSource {
 }
 
 func generateNameFromMountPath(mountPath string) string {
-	s := strings.Trim(mountPath, "/")
-	return strings.ReplaceAll(s, "/", "-")
+	reg, err := regexp.Compile("[^a-zA-Z0-9_-]+")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Burde du ikke returnere her? For nå vil vel bare ting kræsje på linje 216.

Reply:
`log.Error` vil vell avslutte programmet, så er vell heller ikke så heldig. Hva med å heller lage en var på pakkenivå med `regexp.MustCompile`? Så er det en startup check i det minste (som også vil feile tester), og så er det ikke nødvendig å håndtere err i funksjonen.

```go
var regNameNormalizer = regexp.MustCompile("[^a-zA-Z0-9_-]+")
func generateNameFromMountPath(mountPath string) string {
  //...
}
```